PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## What does this PR do?
+
+<!-- MANDATORY -->
```

Comment:
brainstorm: If the PR includes only one patch, this description should be a part of the commit message.

For me, the PR comment is not a part of the git repository. Out of the scope of github.com it does not help / nothing what can be used in distributed development.

The PR comment might give some additional information in the context of this PR issue.   Everything relevant should be in the commit messages.

Reply:
AFAIK one can copy over the PR description to a commit if the PR is merged using the Github UI. Or at least at my day job, we write PR descriptions and then Github move the description to the commit message on merge. See for example: https://github.com/elastic/beats/commit/30f0799743dbcfe636425c317ac48b1bee49ab7d